### ISSUE TITLE
Migrate to SDK-Style project

### ISSUE THREAD

**Issue by virzak** [lsoft/SyncToAsyncExtension#2]
I just noticed that the SyncToAsync.Shared project is non-sdk-style and imports PresentationFramework. Obviously not a bug and makes no difference to the end user, but I was just wondering if this was necessary. SDK-style projects are a lot easier to read and they can produce IL for .NET Framework too.

Same applies to SyncToAsync.Extension.csproj and SyncToAsync.Extension.csproj

**Comment by lsoft** [comment]
@virzak yeah, that is a known pain point)) as I read here ( https://developercommunity.visualstudio.com/t/VSIX-project-with-SDK-style-csproj/1572145 ) support SDK style for VSIX project is not ready now. A hack exists, but I prefer to wait news from MS.

or did I miss something important in this area?

I'l keep this issue open, not to forget.

Thanks!

**Comment by virzak** [comment]
Nope. My bad. Completely forgot that vsix extensions didn't support SDK style projects.
